UsingGoogleSignInButton.md

```diff
@@ -0,0 +1,44 @@
+# Using the Google Sign In Button
+
+Depending upon your platform, GoogleSignIn-iOS provides a few options to use
+when creating a sign-in button.
+
+## SwiftUI
+
+Creating a Google sign-in button in SwiftUI can be as simple as this:
+
+```
+GoogleSignInButton {
+  // Whatever action you'd like to take on tap
+}
+```
+
+This example takes advantage of the initializer's [default argument for the
+view model](GoogleSignInSwift/Sources/GoogleSignInButton.swift#L39).
+The default arguments for the view model will use the light scheme, the
+standard button style, and the normal button state.
+You can supply an instance of [`GoogleSignInButtonViewModel`](GoogleSignInSwift/Sources/GoogleSignInButtonViewModel.swift)
+with different values for these properties to customize the button.
+[This convenience initializer](GoogleSignInSwift/Sources/GoogleSignInButton.swift#L56) 
+provides parameters that you can use to set these values as needed.
+
+## iOS
+
+If you are not using SwiftUI to build your user interfaces, you can either
+create `GIDSignInButton` programmatically, or in a Xib/Storyboard. 
+If you are writing programmatic UI code, it will look something like this:
+
+`GIDSignInButton *button = [[GIDSignInButton alloc] initWithFrame: CGRectMake(<YOUR_RECT>)];`
+
+## macOS
+
+Given that `GIDSignInButton` is implemented as a subclass of `UIControl`, it
```

Comment:
Might want to briefly address `NSHostingView` and the problem it solves here.

Reply:
Good suggestion.